pkg/apis/hco/v1beta1/hyperconverged_webhook.go

```diff
@@ -51,7 +62,7 @@ func (r *HyperConverged) SetupWebhookWithManager(ctx context.Context, mgr ctrl.M
 	whHandler = handler
 	whHandler.Init(hcolog, mgr.GetClient(), operatorNsEnv, isOpenshift)
 
-	certs := []string{filepath.Join(WebhookCertDir, WebhookCertName), filepath.Join(WebhookCertDir, WebhookKeyName)}
+	certs := []string{filepath.Join(GetWebhookCertDir(), WebhookCertName), filepath.Join(GetWebhookCertDir(), WebhookKeyName)}
```

Comment:
Maybe you can call `GetWebhookCertDir` only once, either here or maybe in an `init()` method to a global var?

Reply:
Done